workflows/devnet-build/Jenkinsfile

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/groovy
+properties([
+  parameters([
+    string(defaultValue: "1.0.0-beta.8", description: 'What branch / commit ?', name: 'tag'),
+    string(defaultValue: "devnet", description: 'What network?', name: 'network'),
```

Comment:
We should probably use "tag" so as not to confuse with networks (mainnet, testnet, etc.)